From 60936d96f3dc4819600d1ccca29465a8b0f6323e Mon Sep 17 00:00:00 2001
From: Thejas Nair <thejas@apache.org>
Date: Fri, 6 Feb 2015 19:43:16 +0000
Subject: [PATCH 0021/1164] HIVE-9496 : Slf4j warning in hive command
 (Alexander Pivovarov via Thejas Nair)

git-svn-id: https://svn.apache.org/repos/asf/hive/trunk@1657934 13f79535-47bb-0310-9956-ffa450edef68
(cherry picked from commit 0cd7670e8afa5ed39f230134c8547274d0881188)
---
 jdbc/pom.xml |    4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/jdbc/pom.xml b/jdbc/pom.xml
index 655be72..798356a 100644
--- a/jdbc/pom.xml
+++ b/jdbc/pom.xml
@@ -204,8 +204,10 @@
                   <exclude>org.apache.avro:*</exclude>
                   <exclude>net.sf.opencsv:*</exclude>
                   <exclude>org.antlr:*</exclude>
+                  <exclude>org.slf4j:slf4j-log4j12</exclude>
+                  <exclude>log4j:*</exclude>
                 </excludes>
-              </artifactSet>       
+              </artifactSet>
             </configuration>
           </execution>
         </executions>
-- 
1.7.9.5

